.kitchen.yml

```diff
@@ -1,22 +1,44 @@
+---
 driver:
-  name: vagrant
+  name: dokken
```

Comment:
the dokken config is typically kept in the `kitchen.dokken.yml` files (checkout haproxy or postgresql for a more up to date config)

Reply:
Ok but how can I choose the default driver for my local and circle build ?